src/applications/vaos/tests/services/appointment/index.unit.spec.js

```diff
@@ -73,11 +73,7 @@ describe('VAOS Appointment service', () => {
           startDate,
         ).toISOString()}&end_date=${moment(endDate).toISOString()}&type=va`,
       );
-      expect(global.fetch.secondCall.args[0]).to.contain(
-        `/vaos/v0/appointments?start_date=${moment(
-          startDate,
-        ).toISOString()}&end_date=${moment(endDate).toISOString()}&type=cc`,
-      );
+      expect(global.fetch.secondCall).to.be.null;
```

Comment:
I suppose I could just get rid of this, but now that the appointment fetches are occurring separately this second cc call doesn't occur on error.

Reply:
Reverted now that both calls happen when both vaos service flags are false.